src/session10/homework/vehicle/bicycles/Bicycle.java

```diff
@@ -0,0 +1,32 @@
+package session10.homework.vehicle.bicycles;
+
+public class Bicycle {
+
+    int speed;
+    int gear;
+
+    public Bicycle() {
+        this.speed = 0;
+        this.gear = 1;
+    }
+
+    public void speedUp() {
+        this.speed += 2;
+    }
+
+    public void brake() {
+        if (this.speed > 5) {
```

Comment:
What does this `5` magic number means?
Also, we need `this` keyword on line 18?